Public/Src/Engine/UnitTests/Scheduler/PipExecutorTest.cs

```diff
@@ -1033,6 +1033,94 @@ public async Task FailingProcessWithChunkSizeErrorAndErrorRegex()
                 pathTable => GetConfiguration(pathTable, enableLazyOutputs: false, outputReportingMode: OutputReportingMode.FullOutputOnError));
         }
 
+        [Theory]
+        [InlineData(true, EventId.PipProcessError)]
+        [InlineData(false, EventId.PipProcessError)]
+        [InlineData(false, EventId.PipProcessError, 10 * SandboxedProcessPipExecutor.OutputChunkInLines)]

```

Comment:
>[InlineData(false, EventId.PipProcessError, 10 * SandboxedProcessPipExecutor.OutputChunkInLines)] [](start = 8, length = 97)

What about the case when the regex matches everything but the message gets truncated due to it being too long?